client-angular/package-lock.json

```diff
@@ -0,0 +1,15810 @@
+{
+  "name": "{{pkgName}}",
```

Comment:
Lock file isn't going to get updated by the current upgrade job. I think it's a good idea, but I'd leave it out from this PR - it'll need specific work to support.

Reply:
ok, I removed from angular directory